src/main/java/com/powsybl/openloadflow/network/LfBus.java

```diff
@@ -152,4 +152,8 @@ default boolean isParticipating() {
      * Find bus + parallel branches neighbors.
      */
     Map<LfBus, List<LfBranch>> findNeighbors();
+
+    double getRemoteControlReactivePercent();
```

Comment:
Can you add `voltage` in the name of the method because we also have shunt remote voltage control and reactive remote control.

Reply:
Done